### DISCUSSION TITLE
Comiso

### DISCUSSION THREAD

**Discussion by zoharl3** [libigl/libigl#1945] [Q&A]
I see that the plan is to deprecate comiso:

https://libigl.github.io/changelog/

What's your alternative, e.g. for the quad meshing in the tutorial?

https://libigl.github.io/tutorial/

I'm asking because I'm wondering if there's a better alternative to comiso.

**Comment by jdumas** [comment]
Don't worry libigl 3.0 is not coming out any time soon, so you can keep using what's there for the time being...

**Reply by zoharl3** [reply]
"I'm asking because I'm wondering if there's a better alternative to comiso."
